sql/core/src/test/scala/org/apache/spark/sql/DataFrameAggregateSuite.scala

```diff
@@ -573,4 +578,47 @@ class DataFrameAggregateSuite extends QueryTest with SharedSQLContext {
       spark.sql("SELECT 3 AS c, 4 AS d, SUM(b) FROM testData2 GROUP BY c, d"),
       Seq(Row(3, 4, 9)))
   }
+
+  private def assertNoExceptions(c: Column): Unit = {
```

Comment:
Could you move it to line 566,  just before `test("SPARK-21580 ints in aggregation expressions are taken as group-by ordinal.")`? Thanks!